sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelVersioningTestTest.java

```diff
@@ -1193,6 +1193,29 @@ public void testRequirementR5003NoTestData() {
         assertThrows(NoTestData.class, testClass::testRequirementR5003);
     }
 
+    /**
+     * The old implementation of the test also checked the MdDescriptionVersion and MdStateVersion, although changes to
+     * these values are not visible via reports. The check for these versions were removed and this test is intended
+     * to ensure, that MdDescriptionVersion and MdStateVersion does not affect the test result.
```

Comment:
There seems to be a missing explanation why this test implementation covers MdDescriptionVersion and MdStateVersion, as the test code doesn't transmit or event create them.

Reply:
Changed the description and the unit test to actually set a MdDescriptionVersion